src/openms/source/DATASTRUCTURES/LPWrapper.cpp

```diff
@@ -526,13 +530,11 @@ namespace OpenMS
 #endif
   }
 
-  Int LPWrapper::solve(SolverParam& solver_param, const Size
 #if COINOR_SOLVER == 1
```

Comment:
better to solve with pragma here instead of compile path

Reply:
how?